tools/cmake/utility.cmake

```
@@ -61,10 +87,28 @@ function(set_macro_definitions)
         if(MISSING_REQUIRED_MACROS)
             string(REPLACE ";" ", " MISSING_REQUIRED_MACROS "${MISSING_REQUIRED_MACROS}")
             message("Cannot build ${application_target} because the following required definitions are missing: ${MISSING_REQUIRED_MACROS}")
```

Suggest making the sentence more concise
```suggestion
            message("ERROR: Macros missing definitions for ${application_target}: ${MISSING_REQUIRED_MACROS}")
```